2.13/2.13.4/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.13.4
+Пользователь вводит число. Вывести все числа через запятую от нуля до этого числа, которые деляться на 5 без остатка.
+<script>
+	let i = 1;
+	let number = +prompt('Введите число');
+	let result = '';
+
+	while (i <= number) {
```

Comment:
if we know the iteration quantity, we can use another approach - more simplifier